licenses/LICENSE.tinycolor2

```diff
@@ -0,0 +1,21 @@
+MIT
+Copyright (c), Brian Grinstead, http://briangrinstead.com
+
+Permission is hereby granted, free of charge, to any person obtaining
```

Comment:
what is with all these added licenses?

Reply:
hi, @zentol, thanks for review.

I don't know these licenses. I follow these steps and generated, may be wrong in some place. I will redo these again.
```
Step 1: Rebuild flink
mvn clean install -DskipTests=true -Dfast -T 2C -Dmaven.compile.fork=true -Dhadoop.version=2.7.5 -Dscala.version=2.11.12 -Dscala.binary.version=2.11

Step 2: Run 'tools/releasing/collect_license_files.sh build-target'

Step 3: Replace existing licensing
```

Reply:
@lamber-ken I think @zentol already move these license generation a phase on releasing. Maybe you can redo this patch on current master and I think we don't have such diff now.

Reply:
> @lamber-ken I think @zentol already move these license generation a phase on releasing. Maybe you can redo this patch on current master and I think we don't have such diff now.

Hi, I had rebase lastest patchs and redo these in my local env again, the result is the same as above.